ext/nanopb/nanopb.lb

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2021, Lucas Moesch
+#
+# This file is part of the modm project.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+# -----------------------------------------------------------------------------
+import os
+import subprocess
+import tempfile
+
+from pathlib import Path
+
+def init(module):
+    module.name = ":nanopb"
+    module.description = """
+    # Nanopb - Protocol Buffers for Embedded Systems
+    Nanopb is a small code-size Protocol Buffers implementation in ansi C. It is especially suitable
+    for use in microcontrollers, but fits any memory restricted system.
+    """

```

Comment:
Untab due to this being a raw string (thus preserving the tab in `lbuild discover -n :nanopb`)